src/style.css

```
@@ -1 +1,38 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  box-sizing: border-box;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--5 :nth-child(-n + 5) {
```

```suggestion
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
```
you can shorten this